pom.xml

```diff
@@ -758,6 +758,11 @@
 
     <dependencyManagement>
         <dependencies>
+        	<dependency>
+  				<groupId>org.apache.commons</groupId>
+  				<artifactId>commons-math3</artifactId>
+  				<version>3.6.1</version>
```

Comment:
Version needs to be a property. See other examples in the `pom.xml` where it would be `commons-math3.version`

Reply:
Fixed this in the updated patch